releasenotes/notes/0.18/higher-order-spsa-f2eddb44b00b82e0.yaml

```diff
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    A new optimizer class, :class:`~qiskit.algorithms.optimizers.QNSPSA`,
+    has been added to the :mod:`qiskit.algorithms.optimizers` module. This
+    class implements the
+    `Quantum Natural SPSA (QN-SPSA) <https://arxiv.org/abs/2103.09232>`__
+    algorithm, which a generalization of 2-SPSA algorithm, estimates the
+    Quantum Fisher Information Matrix instead of a the Hessian to obtain a
```

Comment:
```suggestion
    algorithm, a generalization of the 2-SPSA algorithm, estimates the
    Quantum Fisher Information Matrix instead of a the Hessian to obtain a
```